padd.sh

```diff
@@ -1064,15 +1064,20 @@ OutputJSON() {
 }
 
 StartupRoutine(){
-    # Get config variables
+  # Check if PADD is running on a Pi-hole
+  if [ ! -f /etc/pihole/setupVars.conf ]; then
+    printf "%b" "${check_box_bad} Error!\n    PADD is only works in conjunction with Pi-hole!\n"
+    exit 1
+  fi
+  # Get config variables
```

Comment:
Maybe this test could be moved to the beginning of the `main()` function, to exit immediately.

Reply:
Done